core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -464,7 +477,7 @@ class ConsumerBounceTest extends AbstractConsumerTest with Logging {
   private def submitCloseAndValidate(consumer: Consumer[Array[Byte], Array[Byte]],
       closeTimeoutMs: Long, minCloseTimeMs: Option[Long], maxCloseTimeMs: Option[Long]): Future[Any] = {
     executor.submit(() => {
-      val closeGraceTimeMs = 2000
+      val closeGraceTimeMs = 10000
```

Comment:
Not sure if I can in crease this. kraft and kraft+kip848 can't pass this test with 200 closeGraceTimeMs